### PR TITLE
Integer::toString() and Float::toString() are uppercase-string

### PR DESCRIPTION
**Pull request by VincentLanglet**

### CODE REVIEW

File: src/Type/IntegerType.php [hunk 1]
```diff
@@ -83,6 +84,7 @@ public function toString(): Type
 		return new IntersectionType([
 			new StringType(),
 			new AccessoryLowercaseStringType(),
+			new AccessoryUppercaseStringType(),
```
Comment (mvorisek) [inline src/Type/IntegerType.php:87]: FloatType can be improved as well - https://github.com/phpstan/phpstan-src/pull/3613#discussion_r1835336748

Comment (VincentLanglet) [reply src/Type/IntegerType.php:87]: done

**Comment by ondrejmirtes** [comment]
Please send an update to phpstan-doctrine 1.5.x afterwards.

**Comment by ondrejmirtes** [comment]
Thank you.

**Comment by VincentLanglet** [comment]
> Please send an update to phpstan-doctrine 1.5.x afterwards.

Should I target 1.12.x-dev or wait for a release ?

**Comment by ondrejmirtes** [comment]
Just update the require to the next 1.12 release.

**Pull request merged**